.script/tests/KqlvalidationsTests/CustomTables/imWebSession.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
I have created an imWebSession table already since you started. Best if you removed yours from the PR.

Reply:
it is the same file, but I changed the Name to imWebSession